src/lib/Chip/Chip.stories.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="ts">
+  import { Meta, Template, Story } from "@storybook/addon-svelte-csf"
+  import Chip from './Chip.svelte'
+</script>
+
+<Meta title="$lib/Chip" component={Chip} argTypes={{}} />
+
+<Template let:args>
+  <Chip {...args} />
+</Template>
+
+<Story
+  name="default"
+  args={{
+    variant: 'default',
+  }}
+>
+  <Chip variant="default">
+    <span>0x123...456</span>
+  </Chip>
+</Story>
+
+<Story
+  name="info"
+  args={{
+    variant: 'info',
+  }}
+>
+  <Chip variant="info">
+    <span>0x123...456</span>
+  </Chip>
+</Story>
+
+<Story
+  name="success"
+  args={{
+    variant: 'success',
+  }}
+>
+  <Chip variant="success">
+    <span>0x123...456</span>
```

Comment:
Shall we remove these inner `span`s? Given the implementation of `Chip`, it is just a `span` inside another.